### PR TITLE
Honor skip and limit aparameters in action /whisk.system/apimgmt/getA…

### PR DESCRIPTION
**Pull request by larandersson**
## Description
This PR addresses issue (#1692) to honor the skip and limit parameters in the system action  /whisk.system/apimgmt/getApi, which is used by the CLI to obtain a list of managed apis:
```
wsk api list --skip <skip> --limit <limit>
```
## Related issue and scope
- [x] wsk api list command's "--limit" and "--skip" options do not work (#1692)

## My changes affect the following components

- [x] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [x] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [x] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [x] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [ ] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

File: core/routemgmt/common/apigw-utils.js [hunk 1]
```diff
@@ -302,7 +302,7 @@ function getApis(gwInfo, spaceGuid, bpOrApiName) {
   }
   var options = {
     followAllRedirects: true,
-    url: gwInfo.gwUrl+'/'+encodeURIComponent(spaceGuid)+'/apis',
+    url: gwInfo.gwUrl+'/'+encodeURIComponent(spaceGuid)+'/apis?limit='+limit+'&skip='+skip,
```
Comment (mdeuser) [inline core/routemgmt/common/apigw-utils.js]: alas, the `limit` and `skip` query parameters are supported by the openwhisk backend/controller, but not the api gateway
https://github.com/apache/incubator-openwhisk-apigateway/blob/master/scripts/lua/management/routes/apis.lua

Comment (larandersson) [reply core/routemgmt/common/apigw-utils.js]: @mdeuser There's a PR for that too: https://github.com/apache/incubator-openwhisk-apigateway/pull/335, but it's blocked until the build problem is solved.

File: core/routemgmt/getApi/getApi.js [hunk 2]
```diff
@@ -91,7 +93,7 @@ function main(message) {
     gwInfo.gwUrl = message.gwUrlV2;
     gwInfo.gwAuth = message.accesstoken;
     // Obtain the API from the API GW
-    return utils2.getApis(gwInfo, message.spaceguid, message.basepath)
+    return utils2.getApis(gwInfo, message.spaceguid, message.basepath, message.limit, message.skip)
```
Comment (rabbah) [inline core/routemgmt/getApi/getApi.js]: If the values are undefined what’s the behavior of the api call?

Comment (larandersson) [reply core/routemgmt/getApi/getApi.js]: @rabbah The api gateway will ignore them. All apis will be returned.

Comment (rabbah) [reply core/routemgmt/getApi/getApi.js]: thanks.

**Comment by mdeuser** [comment]
@rabbah - yes, i believe it's ok to merge.  the apigw ignores unknown query parameters, so we don't need to worry about a tight sync between the cli and apigw rollout

**Pull request merged**